vendor/views/vendor_admin.py

```diff
@@ -258,11 +284,12 @@ class AdminProfileDetailView(LoginRequiredMixin, DetailView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+        site = settings.SITE_ID
```

Comment:
make this an "else".  Also use the get_current_site() since it will return a Site object (like self.request.site) rather than an integer.

Reply:
Added SetSiteToRequestMixin